packages/descendants/src/useInitDescendants.tsx

```diff
@@ -1,15 +1,22 @@
-import { Dispatch, useReducer } from 'react';
+import { Dispatch, useCallback } from 'react';
+
+import { useStateRef } from '@leafygreen-ui/hooks';
 
 import { DescendantsList } from './Descendants.types';
 import {
   descendantsReducer,
   DescendantsReducerAction,
-  DescendantsReducerType,
 } from './DescendantsReducer';
 
 export interface InitDescendantsReturnObject<T extends HTMLElement> {
+  /** The descendants list in the current render */
   descendants: DescendantsList<T>;
+
+  /** A setter function for the descendants state */
   dispatch: Dispatch<DescendantsReducerAction<T>>;
+
+  /** Accessor function for the most up-to-date descendants list */
+  getDescendants: () => DescendantsList<T>;
```

Comment:
perhaps it makes sense only to return the getter, rather than both?

Reply:
If you want a component to update in response to changing descendants, you'd still need the list object, but we could recommend preferring `getDescendants `.